server/uploadHandler/src/service/dynamodbService.ts

```diff
@@ -1,26 +1,42 @@
 import {
   Converter,
+  GetItemInput,
+  GetItemOutput,
   PutItemInput,
   UpdateItemInput,
 } from "aws-sdk/clients/dynamodb"
+import { table } from "console"
 
-import { File } from "../service/jobModel"
-import { putItem, updateItem } from "./dynamodbClient"
+import { EmailVerification, File, Job } from "../service/jobModel"
+import { getItem, putItem, updateItem } from "./dynamodbClient"
 
 const tableName = process.env.JOB_TABLE as string;
 
+
+export const getJob = async (id:string): Promise<Job> => {
+	const getItemInput: GetItemInput = {
+		TableName:tableName,
+		Key: {id : {S:id}},
+	}
+
+	const ret = await getItem(getItemInput);
+	return Converter.unmarshall(ret.Item!) as Job
+}
+
 export const createNewJobItem = async ( id: string,email: string): Promise<void> => {
 	const newJobItem = {
 		TableName: tableName,
 		Item: {
 			id: { S: id },
 			email: { S: email },
 			createdAt: { S: new Date().toISOString() },
+			emailVerified: {BOOL:false}
```

Comment:
is this consistent with the db schema in the issues? I think it has "uploadTime" instead of "createdAt"

Reply:
oh wait I saw you changed below from "uploadTime" to "CreatedAt" - maybe update the schema in the issues then?